spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/connection/DefaultTcpNioSSLConnectionSupport.java

```diff
@@ -37,9 +38,25 @@
  */
 public class DefaultTcpNioSSLConnectionSupport extends AbstractTcpConnectionSupport implements TcpNioConnectionSupport {
 
-	private volatile SSLContext sslContext;
+	private final SSLContext sslContext;
 
+	private final boolean sslVerifyHost;
+
+	/**
+	 * Create an instance with host verification enabled.
+	 * @param sslContextSupport the ssl context support.
+	 * @since 5.0.8
```

Comment:
I believe this `@since` should go to the other, really new ctor.

If that, I can fix on merge.

Reply:
Oops; yes; thanks.